var/spack/repos/builtin/packages/casacore/package.py

```diff
@@ -66,7 +66,7 @@ class Casacore(CMakePackage):
     depends_on("gsl", when="+dysco")
 
     conflicts("~mpi", when="+adios2")
-    conflicts("+tablelocking", when="+mpi")
+    conflicts("+tablelocking", when="+mpi +adios2")
```

Comment:
https://github.com/casacore/casacore/blob/5a8df94738bdc36be27e695d7b14fe949a1cc2df/CMakeLists.txt#L420-L429 is a sufficient condition to prohibit the table locking option, regardless of the value of the adios2 option. Additionally, casacore allows building against adios2 without MPI enabled (in adios2) https://github.com/casacore/casacore/blob/5a8df94738bdc36be27e695d7b14fe949a1cc2df/CMakeLists.txt#L328-L337 . Not having tried all the combinations, I cannot verify they all actually work in casacore, and perhaps I've missed something, but this change doesn't agree with what's in the CMakeLists.txt file. Is there some particular configuration that you're trying to prohibit that isn't captured by the current conflicts?

Reply:
@pelahi, can you comment on this as you are the author of the change?

Reply:
@pelahi ?

Reply:
@dipietrantonio Are you still interested in the PR? Can you take care of review comments?

Reply:
@alalazo I am withdrawing the PR as it seems this is something we better keep locally. Thank you for your insightful comments nonetheless.